examples/sockets/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # all source files
-set(SRC Sockets.cpp
-        TCP.cpp
-        UDP.cpp)
+set(SRC Sockets.cpp)
 
```

Comment:
Not sure I agree with this change. Why can't we keep this separated for an example?